src/firefly/js/Firefly.js

```diff
@@ -275,7 +276,27 @@ function fireflyInit(props, appSpecificOptions={}, webApiCommands) {
  */
 export function startAsAppFromApi(divId, overrideProps={template: 'FireflySlate'}) {
 
-    const props = {...mergeObjectOnly({...window.firefly.originalAppProps}, overrideProps), div:divId, appFromApi:true};
+
+    const Message = ({}) => (
+        <Stack alignItems='center'>
+            <Typography sx={{fontSize: 'xl4'}} color='neutral'> Firefly Ready </Typography>
```

Comment:
Can we rename it to highlight that this is the python view - something like: "Welcome to Firefly python client"